content/docs/concepts/automated_operations/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Automated Operations
+description: Learn about the core use-case of automated operations.
+weight: 4
+keywords: [keptn, use-cases]
+---
+
+> **Challenge:** In modern microservices environments, you have to deal with systems that can expose unpredictable behavior due to the high number of interdependencies. For example, changing the configuration of one component might have an impact on a different part of the system. Besides, problems evolve and are often dynamic. The nature and impact of a problem can also change drastically over time.
+
+Keptn addresses this challenge by introducing the concept of micro-operations that declare remediation actions for resolving certain problem types or triggering any operational tasks. Micro-operations follow a declarative approach, are atomic building blocks, and get triggered by events.
+
+## Declarative Operations as Code
+
+Keptn complies with a declarative approach for configuring remediation and operations workflows as code on the level of individual microservices (rather than on applications). Consequently, this declaration is versioned next to the operational config and deployed with each new version of the microservice.
+
+Below is an example of a declarative `remediation.yaml` file as used in Keptn. The file defines two problem types and the respective remediation actions. In case of a response time degradation, new instances are scaled up and in the case of a failure rate increase, a feature is disabled. To learn more about the remediation configuration, please continue [here](../../develop/automated_operations/remediation). 
+
+```yaml
+version: 0.2.0
+kind: Remediation
+metadata:
+  name: remediation-service-carts
+spec:
+  remediations:  
+  - problemType: Response time degradation
+    actionsOnOpen:
+    - name: Scaling ReplicaSet by 1
+      description: Scaling the ReplicaSet of a Kubernetes Deployment by 1
+      action: scaling
+      value: 
+        increment: +1
+  - problemType: Failure rate increase
+    - name: Toogle feature flag
+      action: featuretoggle
+      description: Toggle feature flag PromotionCampaign from ON to OFF.
+      value: 
+        PromotionCampaign: off
+```
+
+* This *remediation file* can be interpreted by an automation component and versioned in a Git repository
+
+* This *remediation file* declares what needs to be done and leaves all the details to other components. This approach also follows the operator pattern that is used prominently in Kubernetes.
+
+* The *remediation actions* are defined by the developer for all artifacts (i.e., containers images) that are created. These operations instructions become additional metadata for each artifact.
```

Comment:
Instead of using artifacts, I would say "are defined by the developer for all services that are created". Furthermore, currently, the operations instructions are no metadata for each artifact but only a service.